docs/guides/stay-up-to-date/terraform/how-to-update-to-aws-provider-v4/deployment-walkthrough.md

```diff
@@ -0,0 +1,148 @@
+</tr>
+<tr className="even">
+<td><p>terraform-aws-load-balancer</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-load-balancer/releases/tag/v0.29.0">v0.29.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-data-storage</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-data-storage/releases/tag/v0.24.0">v0.24.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-cache</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-cache/releases/tag/v0.18.0">v0.18.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-messaging</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-messaging/releases/tag/v0.9.0">v0.9.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-static-assets</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-static-assets/releases/tag/v0.15.0">v0.15.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-monitoring</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-monitoring/releases/tag/v0.34.1">v0.34.1</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-openvpn</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-openvpn/releases/tag/v0.24.0">v0.24.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-ecs</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-ecs/releases/tag/v0.34.0">v0.34.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-ci</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-ci/releases/tag/v0.48.0">v0.48.0</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-eks</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-eks/releases/tag/v0.53.0">v0.53.0</a></strong></p></td>
+</tr>
+<tr className="even">
+<td><p>terraform-aws-service-catalog</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.96.1">v0.96.1</a></strong></p></td>
+</tr>
+<tr className="odd">
+<td><p>terraform-aws-cis-service-catalog</p></td>
+<td><p><strong><a href="https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.41.0">v0.41.0</a></strong></p></td>
+</tr>
+</tbody>
+</table>
+
+## Update the Gruntwork Reference Architecture to AWS Provider v4
+
+If you purchased the Gruntwork Reference Architecture, update the relevant code in `infrastructure-live` to use a 
+compatible version of the Gruntwork Infrastructure as Code Library, as per 
+[the compatibility table](#version-compatibility-table) above.
+
+To update your Reference Architecture:
+
+1. Update the underlying module source versions being referenced. 
+    - Use tools like `ripgrep`, `xargs`, and `sed` to accomplish this with ease. 
```

Comment:
NIT: we don't usually recommend using phrases like "simply," "just," or "with ease," as for many users, this stuff isn't easy. Yori posted a videof of a good talk about this in the past.

Reply:
Oh yes, I will update that. Actually that came from my feedback about phrases like "just" and "simply", so it's good that it's coming back around.

Reply:
"Actually" is another one of those words I don't like, either, but I just used it. 🤦

Reply:
Imagine if Terraform provider updates came with patches!